conf/application.conf

```diff
@@ -187,3 +193,11 @@ mongo-lock {
 }
 
 hec-tax-heck-file.default-size = 500
+
+file-notify-config {
```

Comment:
there seems to be a few bits of config related to the file transfer which would be nice if they were collected - maybe something like this:
```
file-transfer{
  extraction-timezone   = "Europe/London"
  extraction-time       = "00:00"
  force-lock-release-after = 5 minutes
  maximum-rows-in-file = 500
  notification-api {
    location = "sdes/stub"
    information-type = ...
    recipient-or-sender = ...
    file-location-base-url = ...
  }
  dispatcher { ... }
}
```
That'd require double checking the app-config-*'s as well though so appreciate that might be annoying to do :grimacing: What do you think?

Reply:
yup, i can do that 👍